README.md

```diff
@@ -5,7 +5,9 @@
 [![Known Vulnerabilities](https://snyk.io/test/github/fastify/fastify-jwt/badge.svg)](https://snyk.io/test/github/fastify/fastify-jwt)
 [![js-standard-style](https://img.shields.io/badge/code%20style-standard-brightgreen.svg?style=flat)](https://standardjs.com/)
 
-JWT utils for Fastify, internally uses [jsonwebtoken](https://github.com/auth0/node-jsonwebtoken).
+JWT utils for Fastify, internally it uses [fast-jwt](https://github.com/nearform/fast-jwt).
+
+**NOTE:** The plugin has been migrated from using `jsonwebtoken` to `fast-jwt`. Even though `fast-jwt` has 1:1 feature implementation with `jsonwebtoken`, some _exotic_ implementations might break. In that case please open an issue with details of your implementation.
```

Comment:
Good catch!

Reply:
Since we are talking about this migration, could we mention a simple upgrade list?

Mainly it concerns the fields name from `issuer` to `iss` etc..

Reply:
Good point.
Examples need an updated to reflect those changes